packages/firestore/test/unit/specs/README.md

```diff
@@ -0,0 +1,58 @@
+# Firestore Spec Tests
+
+The Firestore Spec Tests are a suite of unit tests that fake out external
+sources of events so that they can be completely controlled from the tests
+themselves and verify the correct handling of these events.
+
+There are three sources of external events in the SDK: user writes, input from
+the persistence layer, and input from the watch stream. The spec tests fake all
+of these. Since these external events are asynchronous in nature, in the real
+world there is some non-determinism in the ordering of events. The spec tests
+allow these orderings to be specified explicitly, enabling orderings that are
+difficult or impossible to force when connected to a real backend.
+
+The Firestore Spec Tests are portable and must be kept in sync between the Web
+SDK (this SDK), Android SDK, and iOS SDK. The Web SDK contains the authoritative
+test definitions and they must be manually "exported" to the Android SDK and iOS
+SDK when they change.
+
+## Running the Tests
+
+The spec tests are part of the unit test suite of each SDK. Therefore, to run
+the spec tests simply run the entire unit test suite and the spec tests will be
+executed as a part of it.
+
+For example, in the Web SDK (this SDK), the best way to run the spec tests is
+via the Intellij IDEA IDE. After following the instructions to set up IntelliJ
```

Comment:
99% of JS developers now use VS Code. The other 1% works on Firestore. I think IntelliJ is not that good of a Web IDE unless users pay for the professional edition. I would state that the tests can be run via VS Code and IntelliJ, but mostly we should point out how to use the Node test runner.

I would also give a shout out to the fact that we fake IndexedDb persistence under Node.

Reply:
I ended up just removing this paragraph about IntelliJ since the instructions for running the tests are documented in each repository and don't need to be duplicated here. I also mentioned the IndexedDb persistence faking above.